server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -702,6 +702,18 @@ public void test_rename_subscribed_table_is_not_allowed() throws Exception {
                 "The relation \"doc.t1\" doesn't allow ALTER operations, because it is included in a logical replication subscription.");
     }
 
+    @Test
+    public void test_swap_published_table_is_not_allowed() throws Exception {
```

Comment:
Can this be tested in `SwapTableAnalyzerTest` instead?
Additionally, I don't see a code change in this PR that fixes a LR issue, this is what stated in the changelog entry, but the fix is about not allowing to swap views/foreign tables,etc.

Reply:
> Can this be tested in SwapTableAnalyzerTest instead?

Thank you @matriv, will do that.

>Additionally, I don't see a code change in this PR that fixes a LR issue, this is what stated in the changelog entry, but the fix is about not allowing to swap views/foreign tables,etc.

For published tables, `swap table` was allowed via `PUBLISHED_IN_LOGICAL_REPLICATION` by containining `Operation.ALTER`.
https://github.com/crate/crate/blob/a3ecd974a8d1cd4619e776860edbb01051e0ad6c/server/src/main/java/io/crate/metadata/table/Operation.java#L82-L84
This PR classifies `swap table` as `Operation.ALTER_TABLE_RENAME` instead of `Operation.ALTER` so published tables no longer can be swapped.